### PR TITLE
Internal improvement: Update tests to use the InterfaceAdapter

### PR DESCRIPTION
**Pull request by CruzMolina**
This PR swaps out using `Web3` in scenario & integration tests to properly test the `InterfaceAdapter` instead.

### CODE REVIEW

File: packages/artifactor/test/contracts.js [hunk 1]
```diff
@@ -96,7 +95,7 @@ describe("artifactor + require", () => {
   });
 
   before(() =>
-    web3.eth.getAccounts().then(_accounts => {
+    interfaceAdapter.eth.getAccounts().then(_accounts => {
```
Comment (gnidan) [inline packages/artifactor/test/contracts.js]: Ah, this sort of thing is the culprit! If you want to do this, you should make a `getAccounts` method for real on the interface adapter

File: packages/artifactor/test/contracts.js [hunk 2]
```diff
@@ -230,20 +229,20 @@ describe("artifactor + require", () => {
           "Fallback should not have been triggered yet"
         );
         return example.sendTransaction({
-          value: web3.utils.toWei("1", "ether")
+          value: interfaceAdapter.utils.toWei("1", "ether")
```
Comment (gnidan) [inline packages/artifactor/test/contracts.js]: I think these need to go somewhere else. Use web3 for this for now. The point of the interface adapter is that it removes usage of the Web3.js interface. In places where it doesn't serve to sufficiently replace the Web3.js interface adapter, we should use either the shim or Web3.js itself

File: packages/artifactor/test/contracts.js (hunk 1 above)
Comment (CruzMolina) [reply packages/artifactor/test/contracts.js]: Ahh, so we should have started with `setProvider` then 🤦‍♂ .

**Comment by CruzMolina** [comment]
Closing for now.

**Pull request closed without merging**